docs/_chapters/330-jpms.md

```diff
@@ -127,3 +129,53 @@ public class Main {
 ### Java 8 Support
 
 Bnd's `module-info.class` generation is supported when building with Java 8 or higher. (Yes! I did say Java 8.)
+
+## Advanced Options
+
+There are scenarios where the heuristics used by bnd don't give the desired result because the necessary information is not available or is incorrect.
+
+The `-jpms-module-info-options` instruction provides some capabilities to help the developer handle these scenarios. The instruction uses the _package header syntax_ similar to many other bnd instructions. The keys of these instructions are module names and there are 4 available attributes. They are:
+
+- **`from`** - A module name generated by bnd that should be replaced with the key of the instruction.
+  e.g. 
+  
+  ```properties
+  -jpms-module-info-options: java.enterprise;from="geronimo-jcdi_2.0_spec"
+  ```
+  means that if bnd calculates the module name to be `geronimo-jcdi_2.0_spec` it should replace it with `java.enterprise` 
+  
+- **`ignore`** - If the attribute `ignore="true"` is found the require matching the key of the instruction will not be added
+  e.g. 
+  
+  ```properties
+  -jpms-module-info-options: java.enterprise;ignore="true"
+  ```
+
+  means ignore the module `java.enterprise`
+  
+- **`static`** - If the attribute `static="true"` is found the access of the module matching the key of the instruction will be set to `static`
+  e.g. 
+  
+  ```properties
+  -jpms-module-info-options: java.enterprise;static="true"
+  ```
+
+  means make the `require` for module `java.enterprise` `static`
+  
+- **`transitive`** - If the attribute `transitive="true"` is found the access of the module matching the key of the instruction will be set to `transitive`
```

Comment:
What about `transitive=false` will this prevent bnd from make the require transitive?

Reply:
you're right, I'll handle that for completeness.